confidence_checks/plotting_functions.py

```diff
@@ -0,0 +1,77 @@
+from typing import Tuple
+from matplotlib import pyplot as plt
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+from models import model_interface
+
+
+def plot_model_confidence(
```

Comment:
Will these plots be used for anything beyond confidence checking? It may be worth moving these into a visualization subdir or otherwise have all visualizaation/plotting tools together instead of split out into different subdirs.

Reply:
Currently they are only used for confidence checking. I expect to modify them for more general use later. The biggest change in that modification will be to add support for visualizing high dimensional data.